rpcs3/rpcs3qt/patch_manager_dialog.cpp

```diff
@@ -135,9 +136,25 @@ void patch_manager_dialog::load_patches()
 	path_list << QDir(QString::fromStdString(patches_path)).entryList(filters);
 	path_list.removeDuplicates(); // make sure to load patch.yml and imported_patch.yml only once
 
+	bool has_errors = false;
+
 	for (const auto& path : path_list)
 	{
-		patch_engine::load(m_map, patches_path + path.toStdString());
+		if (!patch_engine::load(m_map, patches_path + path.toStdString()))
+		{
+			has_errors = true;
+		}
+	}
+
+	if (show_error && has_errors)
+	{
+		// Open a warning dialog after the patch manager was opened
+		QTimer::singleShot(100, [this]()
+		{
+			QMessageBox::warning(this, tr("Incompatible patches detected"),
+				tr("Some of your patches are not compatible with the current version of RPCS3's Patch Manager.\n\nMake sure that all the patches located in \"%0\" contain the proper formatting that is required for the Patch Manager Version %1.")
+				.arg(QString::fromStdString(patch_engine::get_patches_path())).arg(QString::fromStdString(patch_engine_version)));
```

Comment:
You have a const a bit before, so don't need to call get_patches_path()
```suggestion
				.arg(QString::fromStdString(patches_path)).arg(QString::fromStdString(patch_engine_version)));
```

Reply:
Ah thx. I had some extra variable set to replace my name in the path and forgot to reuse the proper one in the final version.

Reply:
Can you also make a suggestion for the capture? I'm on the train and can't properly work on this on my own right now